site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+          meets user needs and expectations.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Pay attention to label lengths.</strong>
+      <ul style="margin-top:0;">
+        <li>Keep the as concise as possible.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Be responsive.</strong>
+      <ul style="margin-top:0;">
+        <li>Make sure the navigation pattern works well on different devices and screen sizes.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Prioritize important elements.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Organize the navigation items hierarchically, highlighting the most important elements and grouping similar
+          items together.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Customize responsibly.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          When customizing the navigation pattern, ensure that the changes do not negatively impact usability,
+          accessibility, or consistency.
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+<div
+  class="guideline-dont"
+  style="background: var(--color-error-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);">
+    <IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Don't</strong>
+  </div>
+  <ul style="margin-bottom:0;">
+    <li>
+      <strong>Don't sacrifice clarity for style.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          While it's essential to maintain a visually appealing design, prioritize clarity and usability when
+          implementing the navigation pattern.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't disregard performance.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Optimize the navigation pattern for performance by reducing file sizes, minimizing HTTP requests, and using
+          efficient code.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't neglect mobile users.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Do not assume all users are on desktop devices; ensure the navigation pattern works well on mobile devices and
+          touchscreens.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't ignore feedback.</strong>
+      <ul style="margin-top:0;">
+        <li>Do not disregard user feedback or testing results when refining and improving the navigation pattern.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't overcomplicate.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Avoid adding unnecessary elements, features, or interactions to the navigation pattern, which may confuse or
+          overwhelm users.
```

Comment:
Should there also be something about don't add all the same links in navigation to content area e.g. linkboxes?

Reply:
Quite many of these are actually more general than just navigation related. Should we have a separate page "do user testing and gather feedback" and make these more specifically to navigation?